controllers/apply/form-router-next/index.js

```diff
@@ -152,9 +78,76 @@ function initFormRouter({
                 });
             }
         }),
-        handleMultipartFormData,
-        injectCopy('applyNext'),
-        setCommonLocals,
+        function(req, res, next) {
+            function needsMultipart() {
```

Comment:
Could you reinstate the comment that went with this? It's not always obvious why we're doing this stuff so would be worth keeping it in to explain the *why*

Reply:
Good shout, didn't mean to loose that